### ISSUE TITLE
Selecting image in Background dialog shows error with file in a folder

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#1517]
Running **Properties part 5** test, section "background-image on a DIV":

While working in a **file inside of a folder** (one level deep), opened the **Background dialog**, selected "image" as "background-image type", then clicked on URL's "…" button to select SampleBanner.jpg.  URL is correct ("../SampleBanner.jpg"), but console shows an **error trying to load "http://localhost:50000/SampleBanner.jpg"**.

@JonFerraiolo

**Comment by JonFerraiolo** [comment]
@childsb

**Comment by JonFerraiolo** [comment]
The URL appears in the console as `/SampleBanner.jpg`. The page editor does show the correct background and preview-in-browser shows the correct background.

**Comment by jhpedemonte** [comment]
Correct, forgot to add that.  However, if you then go and play with the options below the URL line, each time you change an option, you'll see the load failure again.

Why is the background image loaded by the dialog anyway?  Is it even necessary?

**Comment by aerwin** [comment]
I noticed this one today as well. I don't think the html file being edited has to be inside a folder (e.g., I'm seeing it with a top-level html file). 

I investigated a bit because at first I was worried I caused it (though now obviously see it's been on the books awhile)... 

The dialog is trying to put the image in the little preview box it owns. Inspecting the element for the preview box in Chrome, the background image is set the same way as it it in the HTML source:

> background-image: url('SampleBanner.jpg')

So, I'm suspect the relative path from the perspective of the dialog is different from relative path from the persective of the HTML file..
